apollo-idling-resource/src/main/java/com/apollographql/apollo/espresso/ApolloIdlingResource.java

```diff
@@ -0,0 +1,61 @@
+package com.apollographql.apollo.espresso;
+
+
+import android.support.test.espresso.IdlingResource;
+
+import com.apollographql.apollo.ApolloClient;
+import com.apollographql.apollo.IdleCallback;
+
+import javax.annotation.Nonnull;
+
+import static com.apollographql.apollo.api.internal.Utils.checkNotNull;
+
+/**
+ * An Espresso {@link IdlingResource} for {@link com.apollographql.apollo.ApolloClient}.
+ */
+public class ApolloIdlingResource implements IdlingResource {
```

Comment:
Hmm, we can move it to a separate repo which is a part of the apollo-graphql project, probably because it's a good to have feature, but not an absolutely necessary one. Is that what you also think? I don't know what others feel though.